recipes/openfoam/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "openfoam" %}
+{% set version = "v2412" %}
```

Comment:
```suggestion
{% set version = "2412" %}
```
Don't include the v in the conda version.

Reply:
Removed the "vé as suggested